test/tools/ossfuzz/strictasm_diff_ossfuzz.cpp

```diff
@@ -80,6 +80,7 @@ extern "C" int LLVMFuzzerTestOneInput(uint8_t const* _data, size_t _size)
 	{
 		yulFuzzerUtil::interpret(
 			os1,
+			*EVMDialect::strictAssemblyForEVMObjects(langutil::EVMVersion()),
```

Comment:
Dialect is the third argument going by the definition of interpret. So please change to 

```
		yulFuzzerUtil::interpret(
			os1,
            stack.parserResult()->code,
			*EVMDialect::strictAssemblyForEVMObjects(langutil::EVMVersion()),
...
```

Reply:
oh shoot!